src/Params/EnvelopeParams.h

```diff
@@ -19,6 +19,14 @@
 #include "Presets.h"
 
 namespace zyn {
+    
+   enum envmode_enum {
+        ADSR_lin,
```

Comment:
Looks like an off by one.
Since these were previously integer values saved to disk it seems reasonable to define the values explicitly. Perhaps I'm mistaken, but it looks like the values of the new enum do not match the old values.